hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AbfsStatistic.java

```diff
@@ -57,7 +57,23 @@
   FILES_DELETED("files_deleted",
       "Total number of files deleted from the object store."),
   ERROR_IGNORED("error_ignored",
-      "Errors caught and ignored.");
+      "Errors caught and ignored."),
+
+  //Network statistics.
+  CONNECTIONS_MADE("connections_made",
+      "Total number of times connection was made with Data store."),
+  SEND_REQUESTS("send_requests",
+      "Total number of times http requests was sent to the data store."),
+  GET_RESPONSE("get_response",
+      "Total number of times response was recorded after sending requests."),
+  BYTES_SEND("bytes_send",
+      "Total bytes sent through http requests."),
```

Comment:
how about "bytes sent from Azure Datalake"

Reply:
shouldn't it be "bytes sent to Azure Datalake"?

Reply:
yes. Or just "bytes uploaded". avoids having to deal with any future rebranding